pkg/sloop/store/typed/relationshiptable_test.go

```diff
@@ -0,0 +1,35 @@
+package typed
+
+import (
+	"github.com/salesforce/sloop/pkg/sloop/store/untyped"
+	"github.com/stretchr/testify/assert"
+	"testing"
+	"time"
+)
+
+
+func Test_RelationshipTableKey_OutputCorrect(t *testing.T) {
+	untyped.TestHookSetPartitionDuration(time.Hour)
+	k := NewRelationshipKey(someTs, someKind, someNamespace, someName, someUid)
+	assert.Equal(t, "/relation/001546398000/somekind/somenamespace/somename/68510937-4ffc-11e9-8e26-1418775557c8", k.String())
+}
+
+func Test_RelationshipTableKey_ParseCorrect(t *testing.T) {
+	untyped.TestHookSetPartitionDuration(time.Hour)
+	k := &RelationshipKey{}
+	err := k.Parse("/relation/001546398000/somekind/somenamespace/somename/68510937-4ffc-11e9-8e26-1418775557c8")
```

Comment:
consider building the key using the literals instead of duplicating the literals

Reply:
or add and use:
```
const (
    partitionId = 001546398000
)
var (
   tableKey := fmt.Sprintf("/%s/%s/%s/%s/%s/%s", relationshipTableName, partitionId, someKind, someNamespace, someName, someUid)
)
```

Reply:
updated